### PR TITLE
add UI for language selection (contributed by Grise3)

### PR DESCRIPTION
**Pull request by sweet235**
This was contributed by Grise3. Add a graphical user interface to select the language.

### CODE REVIEW

**Comment by illwieckz** [comment]
We may autogenerate the list at some point in the future but that looks good enough for a start.

**Comment by sweet235** [comment]
I should add that Grise3 asked us to change whatever we see fit, no need to ask them.

**Comment by sweet235** [comment]
> We already have a language datasource. Does this not work?

How is it supposed to work? I am guilty of using English on my machines, for the ease of googling problems.

**Comment by bmorel** [comment]
This needs to be merged. Current way is to know about the translation and to know about the console cvar. Those changes are an obvious improvement, and even if they are not perfect, improvements should be merged.

> We already have a language datasource. Does this not work?

A really good question. I think not many persons know how to deal with that (outdated) rmlui thing.

File: pkg/unvanquished_src.dpkdir/ui/options_ui.rml [hunk 1]
```diff
@@ -4,8 +4,21 @@
 		<link type="text/template" href="/ui/shared/window.rml" />
 	</head>
 	<body id="options_ui" template="window" style="width: 32em; margin: 1em;">
-		<h1><translate>User interface</translate></h1>
-
+		<row>
+			<h1><translate>User interface</translate></h1>
+			<h3><translate> Language</translate> </h3>
```
Comment (slipher) [inline pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: Remove leading spaces inside of `<translate>` tags

**Comment by DolceTriade** [comment]
It's not dark magic or anything...

Instead of the dropdown you would use:

```
<dataselect source="languages.default" fields="language" cvar="language" />
```

File: pkg/unvanquished_src.dpkdir/ui/options_ui.rml (hunk 1 above)
Comment (sweet235) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: Who are you talking to?

Comment (illwieckz) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: Actually that leading space will be included in the translation string, this leading space is not ignored.

Comment (sweet235) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: Sure. Either fix it or don't, do whatever you want. But attempts to command me to work are completely inappropriate.

Comment (illwieckz) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: I agree.

Comment (slipher) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: The PR is pointed at a branch on your personal repository so no one else can edit it.

Comment (sweet235) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: Does "Allow edits by maintainers" not work?

Comment (slipher) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml:9]: I believe that refers to the messages in the comment thread. Not the code.

Comment (slipher) [reply pkg/unvanquished_src.dpkdir/ui/options_ui.rml]: Done by me

**Comment by slipher** [comment]
LGTM

**Pull request merged**